iet.so/src/iet_worker.cpp

```diff
@@ -335,3 +343,23 @@ void IETWorker::run() {
 
     sleep(5);
 }
+
+/**
+ * @brief blas callback function upon gemm operation completion
+ * @param status gemm operation status
+ * @param user_data user data set
+ */
+void IETWorker::blas_callback (bool status, void *user_data) {
```

Comment:
same as for GST function, can this be common?

Reply:
At rvs_blas level, we have common callback function rvs_blas::hip_stream_callback which call the corresponding worker callbacks. In worker callback, we unlock worker thread and set the blas operation status.